untie/src/untie.cpp

```diff
@@ -0,0 +1,19 @@
+
+#include "untie.h"
+#include <iostream>
+
+namespace knot {
+
+Untie::Untie() {
+    m_engine = std::make_unique<knot::Engine>();
+    m_window = m_engine->get_window_module();
+}
+
+void Untie::run() {
+    std::cout << "here" << std::endl;
```

Comment:
`log::debug("Here");` and I need to make a PR to fix this. (So debug messages appear by default in Debug mode.)

Reply:
I meant ot remove this cout c: